src/views/batch/components/batch-details/batch-details.view.jsx

```diff
@@ -13,6 +13,9 @@ import Col from '../../../shared/col/col'
 function BatchDetails ({ batch }) {
   const classes = useBatchDetailsStyles()
   const [areDeailsVisible, setDetailsVisible] = React.useState()
+  const timeZoneOffsetInHours = -(new Date().getTimezoneOffset() / 60)
+  const timeZoneOffset = timeZoneOffsetInHours >= 0 ? '+' + timeZoneOffsetInHours : timeZoneOffsetInHours
```

Comment:
Just a small suggestion here just to keep things cleaner:

What do you think if me rename the `src/utils/time-converter.js` to a more generic `src/utils/date.js`and we move the calculation of this offset with the `+` or `-` sign to a function there?